Strategic-Initiatives.md

```diff
@@ -42,6 +41,7 @@ and have the support needed.
 | Workers             | Anna Henningsen      | https://github.com/nodejs/worker                |
 | N-API               | Michael Dawson       | https://github.com/nodejs/abi-stable-node       |
 | Open Web Standards  | Myles Borins Joyee Cheung | https://github.com/nodejs/open-standards   |
+| Modules             | Myles Borins.        | https://github.com/nodejs/modules               |
```

Comment:
Maybe you were trying to underline it with the period 😆 
```suggestion
| Modules             | Myles Borins         | https://github.com/nodejs/modules               |
```

Reply:
Myles Borins<sub>🎤 </sub>